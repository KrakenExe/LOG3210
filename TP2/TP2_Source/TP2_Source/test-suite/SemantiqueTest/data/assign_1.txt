Declaration
bool a;
bool b;
EndDeclaration

a = 1;
b = true;
