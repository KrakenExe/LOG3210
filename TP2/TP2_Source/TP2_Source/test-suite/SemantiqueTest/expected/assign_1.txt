Invalid type in assignation of Identifier a... was expecting bool but got num